CCDB/src/BasicCCDBManager.cxx

```diff
@@ -43,5 +43,30 @@ std::pair<uint64_t, uint64_t> CCDBManagerInstance::getRunDuration(int runnumber)
   return std::make_pair(sor, eor);
 }
 
+std::string CCDBManagerInstance::getSummaryString() const
+{
+  std::string res = fmt::format("{} queries", mQueries);
+  if (mCachingEnabled) {
+    res += fmt::format(" for {} objects", mCache.size());
+  }
+  res += fmt::format(", {} good fetches (and {} failed ones", mFetches, mFailures);
+  if (mCachingEnabled && mFailures) {
+    int nfailObj = 0;
+    for (const auto& obj : mCache) {
+      if (obj.second.failures) {
+        nfailObj++;
+      }
+    }
+    res += fmt::format(" for {} objects", nfailObj);
+  }
+  res += fmt::format("), instance: {}", mCCDBAccessor.getUniqueAgentID());
+  return res;
+}
+
+void CCDBManagerInstance::endOfStream()
+{
+  LOG(info) << "CCDB summary: " << getSummaryString();
```

Comment:
will this function be called from somewhere automatically?

Reply:
I would maybe be more specific and say "CCDBManager summary" instead of CCDB summary (which may give a false impression).

Reply:
@sawenzel yes, from the analyses: https://github.com/AliceO2Group/AliceO2/pull/9713.
OK for the log.